things/lift/lift.tm.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context":[
+     "https://www.w3.org/2019/wot/td/v1",
+     "https://www.w3.org/2022/wot/td/v1.1",
+     {
+        "@language":"en"
+     }
+  ],
+  "@type":"tm:ThingModel",
+  "title":"{{THING_NAME}}",
+  "description":"Lift Thing",
+  "securityDefinitions":{
+      "nosec_sc":{
+        "scheme":"nosec"
+     }
+  },
+  "security":[
+     "nosec_sc"
+  ],
+  "base": "{{PROTOCOL}}://{{HOSTNAME}}:{{PORT_NUMBER}}/",
+  "properties":{
+      "lightSwitch": {
+         "type": "boolean",
+         "readOnly": false,
+         "writeOnly": false,
+         "observable": true
+      },
+      "onTheMove": {
+         "type": "boolean",
+         "readOnly": true,
+         "writeOnly": false,
+         "observable": true
+      },
+      "floorNumber": {
+         "type": "integer",
```

Comment:
we need minimum maximum for this

Reply:
Fixed at [1e7bc99](https://github.com/eclipse-thingweb/test-things/pull/9/commits/1e7bc99a0e815868f9254edfff03ef9562635cd2)